SmartTutor/Controllers/ContentRepositoryService.cs

```diff
@@ -0,0 +1,129 @@
+﻿using SmartTutor.ContentModel;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SmartTutor.Controllers
+{
+    public class ContentRepositoryService
+    {
+        public ContentRepositoryService() { }
+
+        public EducationContent GetLongParameterListContent()
+        {
+            EducationContent educationContent = new EducationContent();
+
+            educationContent.EducationQuality = 4;
+            educationContent.EducationDifficulty = 4;
+            educationContent.EducationSnippets = new List<EducationSnippet>();
+
+            EducationSnippet snippetOne = new EducationSnippet();
+            EducationSnippet snippetTwo = new EducationSnippet();
+            EducationSnippet snippetThree = new EducationSnippet();
+            EducationSnippet snippetFour = new EducationSnippet();
+            EducationSnippet snippetFive = new EducationSnippet();
+
+            snippetOne.SnippetQuality = 4;
+            snippetOne.Tags = new List<Tag>(); snippetOne.Tags.Add(Tag.MustKnow);
+            snippetOne.EducationType = EducationType.ShortText;
+            snippetOne.SnippetDifficulty = 3;
+            snippetOne.Content = "This method have problem with long parameter list";
+            
+
```

Comment:
In general, there is no need for two blank lines, neither inside nor outside a function.